src/en/charms.md

```diff
@@ -1,4 +1,5 @@
 # Introducing Charms
+## Missing Words
```

Comment:
Me too. I may have put it in there while testing watches or whatnot. I'll remove - good catch!